src/gnn_reco/components/loss_functions.py

```diff
@@ -0,0 +1,30 @@
+import torch
+import numpy as np
+from torch.nn import CrossEntropyLoss
+
+
+def LogCosh(prediction, graph, target):
+    return torch.sum(torch.log(torch.cosh(((prediction[:,0]-graph[target])))))
+
+def CustomCrossEntropyLoss(prediction, graph, target):
+    f = CrossEntropyLoss()
+    return f(prediction,graph[target].long())
+
+
+def VonMisesSineCosineLoss(prediction, graph, target):
```

Comment:
This function seems to make non-trivial assumptions about the structure of `prediction`. It might be beneficial to have a docstring that detail these assumptions?

Reply:
doc string added